### ISSUE TITLE
Unable to connect to FreeNAS API?

### ISSUE THREAD

**Issue by wwilson603** [TheGrandWazoo/freenas-proxmox#44]
When trying to create a VM disk the following error occurs:

`TASK ERROR: unable to create VM 100 - error with cfs lock 'storage-vmstorage_iscsi': Unable to connect to the FreeNAS API service at '192.168.77.2' using the 'http' protocol at /usr/share/perl5/PVE/Storage/LunCmd/FreeNAS.pm line 241.`

The disk name does appear in the FreeNAS tree though:
![2019-07-12 04_41_37-bsd-nas-01 - FreeNAS-11 2-U5 (c129415c52)](https://user-images.githubusercontent.com/52811963/61125729-7303fb80-a45f-11e9-9b7e-b4736d3dcaf9.png)

Setup is as follows:
- DAC 10GbE (Intel X520) connection running between the two servers, with a static IP set on each end
- FreeNAS 11.2-U5
- ProxMox 5.4-3 (config for ZFS connection below)
![2019-07-11 22_12_56-prxmx-vhost-01 - Proxmox Virtual Environment](https://user-images.githubusercontent.com/52811963/61104124-ec80f700-a429-11e9-9d3f-5c10c0bd5413.png)

**Comment by TheGrandWazoo** [comment]
Make sure you FreeNAS HTTP/HTTPS bind address is for this interface. If you are using like a SAN or separate network then the Management. The API must have HTTP/HTTPS enabled on this interface.

**Comment by wwilson603** [comment]
Thanks for the pointer! I was able to yet again figure it out. I always seem to be running into these small quirks here and there that stops my progress.

I did it by changing this setting in FreeNAS to the wildcard value:
![2019-07-12 12_27_19-Window](https://user-images.githubusercontent.com/52811963/61153688-d8c4a780-a4a0-11e9-8c52-31eb9e521baf.png)

it's a bit un-intuitive that it mentions specifically the "WebGUI" and due to that I would have never thought to have touch that setting, but all is working now!

Thanks so much for your work and support!
